packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4256,20 +4256,46 @@ class RenderFollowerLayer extends RenderProxyBox {
 ///   * [Layer.find], for an example of how this value is retrieved.
 ///   * [AnnotatedRegionLayer], the layer this render object creates.
 class AnnotatedRegionRenderObject<T> extends RenderProxyBox {
+
+  /// Creates a new [AnnotatedRegionRenderObject] to insert [value] into the
+  /// layer tree.
+  ///
+  /// If [sized] is true, the layer is provided with the size of this render
+  /// object to clip the results of [Layer.findRegion].
+  ///
+  /// Neither [value] nor [sized] can be null.
+  AnnotatedRegionRenderObject(T value, bool sized)
```

Comment:
btw the naming scheme is usually RenderFoo for the render object, and Foo for the widget.

Reply:
Fixed